src/pages/App.tsx

```diff
@@ -80,97 +90,108 @@ export default function App(): ReactElement {
   }, [boot])
 
   return (
-    <Suspense fallback={null}>
-      <Web3ReactManager>
-        <MinichefProvider>
-          <GaugeProvider>
-            <TokensProvider>
-              <ExpandedPoolsProvider>
-                <UserStateProvider>
-                  <PricesAndVoteData>
-                    <PendingSwapsProvider>
-                      <AprsProvider>
-                        <RewardsBalancesProvider>
-                          <LocalizationProvider dateAdapter={AdapterDateFns}>
-                            <AppContainer>
-                              <TopMenu />
-                              <Switch>
-                                <Route exact path="/" component={Swap} />
-                                <Route exact path="/pools" component={Pools} />
-                                {pools.map((pool) => (
+    <QueryClientProvider client={queryClient}>
+      <ReactQueryDevtools initialIsOpen />
```

Comment:
Can we include this only in dev environments?

Reply:
It automatically does that.